src/SprykerEco/Shared/Econda/EcondaConstants.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Shared\Econda;
+
+interface EcondaConstants
+{
+
+    const ECONDA_CSV_DELIMITER = "|";
+    const ECONDA_CSV_CATEGORY_DELIMITER = "^^";
```

Comment:
Same here

Reply:
ECONDA_CSV_DELIMITER and ECONDA_CSV_CATEGORY_DELIMITER are also defined in Zed\Econda\EcondaConfig. Actually those are the ones that are used. So I will remove both from EcondaConstants.php. Is it OK to have them in Zed\Econda\EcondaConfig or should I move them to Shared\Econda\EcondaConfig?

Reply:
If only ever needed in Zed thats totally fine.